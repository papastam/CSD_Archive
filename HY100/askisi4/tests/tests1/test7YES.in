7
22 31 40 49 2 11 20 
21 23 32 41 43 3 12 
13 15 24 33 42 44 4 
5 14 16 25 34 36 45 
46 6 8 17 26 35 37 
38 47 7 9 18 27 29 
30 39 48 1 10 19 28 
