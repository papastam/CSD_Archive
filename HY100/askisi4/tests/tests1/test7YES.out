ROW 1 YES
ROW 2 YES
ROW 3 YES
ROW 4 YES
ROW 5 YES
ROW 6 YES
ROW 7 YES
COLUMN 1 YES
COLUMN 2 YES
COLUMN 3 YES
COLUMN 4 YES
COLUMN 5 YES
COLUMN 6 YES
COLUMN 7 YES
DIAG1 YES
DIAG2 YES
UNIQUE YES
MAGIC YES
